datafusion/sql/src/unparser/dialect.rs

```diff
@@ -108,6 +113,15 @@ pub trait Dialect: Send + Sync {
     fn supports_column_alias_in_table_alias(&self) -> bool {
         true
     }
+
+    fn scalar_function_to_sql_overrides(
```

Comment:
👍 

Could you please add documentation here explaining what this function does / is used for?